src/popup/components/TransactionList.vue

```diff
@@ -140,7 +140,7 @@ export default defineComponent({
         (transaction) => (
           !searchPhrase.value
           || includesCaseInsensitive(
-            getTxSymbol.value(transaction),
+            getTxSymbol(transaction as ITransaction),
```

Comment:
Instead of using `as` here I suggest to allow to pass `ICommonTransaction` as the argument of `getTxSymbol`.